### PR TITLE
[docs] Update dashboard documentation

### PR DESCRIPTION
**Pull request by RomuDeuxfois**

### CODE REVIEW

File: docs/usage/evaluate/overview.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+snapshot of your global security posture. Below is a breakdown of the various widgets available on this page.
+
+## Metric cards
+
+This section displays the count of different components within the platform, shown in two metrics:
+
+- The first metric represents the number of objects created over the last 180 days.
+- The second metric reflects the number of objects created in the last 30 days.
+
+![metric cards](assets/metric-cards.png)
```
Comment (EllynBsc) [inline docs/usage/evaluate/overview.md:17]: Here @RomuDeuxfois, we have 4 metrics in 4 different cards no ? Not clear to me
- Scenarios
- Simulations
- Players
- Assets

File: docs/usage/getting-started.md [hunk 2]
```diff
@@ -4,9 +4,14 @@
...
 
-This guide aims to give you a full overview of the OpenBAS features and workflows. The platform can be used in various contexts to handle Breach and Attack simulations at technical or strategical levels. OpenBAS has been designed as a part of the Filigran XTM suite and can be integrated with [OpenCTI](https://filigran.io/solutions/open-cti/) to generate meaningful attack scenario based on real threat. OpenBAS is result-oriented with many dashboard helping you to evaluate you securoty posture given a defined context.
+This guide aims to give you a full overview of the OpenBAS features and workflows. The platform can be used in various
+contexts to handle Breach and Attack simulations at technical or strategical levels. OpenBAS has been designed as a part
+of the Filigran XTM suite and can be integrated with [OpenCTI](https://filigran.io/solutions/open-cti/) to generate
+meaningful attack scenario based on real threat. OpenBAS is result-oriented with many dashboard helping you to evaluate
```
Comment (EllynBsc) [inline docs/usage/getting-started.md:10]:
```suggestion
meaningful attack scenarios based on real threat. OpenBAS is result-oriented with many dashboards helping you to evaluate
```

File: docs/usage/getting-started.md [hunk 3]
```diff
@@ -15,49 +20,68 @@ Here are some examples of use cases:
 
 ## Home screen
 
-The Home screen gives any visitor on the OpenBAS platform an outlook on the live of the platform as well as an overview of your global security posture.
-
+The Home screen gives any visitor on the OpenBAS platform an outlook on the live of the platform as well as an overview
```
Comment (EllynBsc) [inline docs/usage/getting-started.md:23]:
```suggestion
The Home screen provides every OpenBAS platform visitor with a snapshot of the platform activity as well as an overview 
```

File: docs/usage/getting-started.md [hunk 4]
```diff
@@ -15,49 +20,68 @@ Here are some examples of use cases:
...
-If you want to simulate attack at a technical level, you will need to install a Simulation Agent that will play technical events on your imported assets. To do so, you can go to the dedicated panel from the top right button and follow installation instructions. By default, Caldera is part of the OpenBAS stack.
+
+If you want to simulate attack at a technical level, you will need to install a Simulation Agent that will play
+technical events on your imported assets. To do so, you can go to the dedicated panel from the top right button and
+follow installation instructions. By default, Caldera is part of the OpenBAS stack.
 
 ![Simulation Agents screen](assets/simulation_agents_screen.png)
 
-The [Caldera agent](inject-caldera.md) will allow you to play various technical events, based on the Mitre ATT&CK matrix.
+The [Caldera agent](inject-caldera.md) will allow you to play various technical events, based on the Mitre ATT&CK
```
Comment (EllynBsc) [inline docs/usage/getting-started.md:41]: Here should we mention the Mitre ATT&CK doc  - https://attack.mitre.org/

File: docs/usage/getting-started.md [hunk 5]
```diff
@@ -15,49 +20,68 @@ Here are some examples of use cases:
...
+matrix.
 
 ### Building your Scenario
-Once integrations is done, you are ready to create your first Scenario! 
 
-[Scenarios](scenario.md) act as template for your Breach and Attack simulations. After establihing such a template, you will be able to schedule it as a one shot simulation, or as a recurring one. 
+Once integrations is done, you are ready to create your first Scenario!
+
+[Scenarios](scenario.md) act as template for your Breach and Attack simulations. After establihing such a template, you
```
Comment (EllynBsc) [inline docs/usage/getting-started.md:48]:
```suggestion
[Scenarios](scenario.md) act as template for your Breach and Attack simulations. After establishing such a template, you
```

File: docs/usage/getting-started.md [hunk 6]
```diff
@@ -15,49 +20,68 @@ Here are some examples of use cases:
...
+  Scenario. If you want to stay strategical, you can select inject like "Send individual mails". If you want to go
+  technical, you can select injects linked to attack pattern (Caldera integration allows you to play hundreds of them).
+- Then, define [who or what will be targeted](targets.md) by those injects, customize them, and define what is expected
+  to happen. For example, you expect the targeted team to perform a specific action and the animation team will
+  validated this expectation manually. Or, you expect the technical event to be prevented and it will be automatically
+  checked through your integrations with your security systems.
 - Do not forget to define when the inject is played in the scenario chronology.
 
-Optionnaly, you can buff up your scenario by adding [Documents](components/documents.md), [Media pressures](components/media_pressure.md), or even CTF [Challenges](components/challenges.md) to your injects.
+Optionnaly, you can buff up your scenario by
```
Comment (EllynBsc) [inline docs/usage/getting-started.md:63]:
```suggestion
Optionally, you can enhance your scenario by
```

File: docs/usage/getting-started.md [hunk 7]
```diff
@@ -15,49 +20,68 @@ Here are some examples of use cases:
...
+CTF [Challenges](components/challenges.md) to your injects.
 
 ### Play the simulation
 
-You can now schedule your [Simulation](simulation.md) by hitting the blue "Simulate now" button. Choose your moment and hit start.
+You can now schedule your [Simulation](simulation.md) by hitting the blue "Simulate now" button. Choose your moment and
+hit start.
 
-On time, a Simulation based on your Scenario template is generated. it is listed in your Scenario overview and in the Simulations menu. Fron there, you can follow the course of the Simulation and interact with it, for example to validate manual expectations.
+On time, a Simulation based on your Scenario template is generated. it is listed in your Scenario overview and in the
```
Comment (EllynBsc) [inline docs/usage/getting-started.md:72]:
```suggestion
On time, a Simulation based on your Scenario template is generated. It is listed in your Scenario overview and in the
```

File: docs/usage/getting-started.md [hunk 8]
```diff
@@ -15,49 +20,68 @@ Here are some examples of use cases:
...
+Simulations menu. Fron there, you can follow the course of the Simulation and interact with it, for example to validate
```
Comment (EllynBsc) [inline docs/usage/getting-started.md:73]:
```suggestion
Simulations menu. From there, you can follow the course of the Simulation and interact with it, for example to validate
```

**Review by EllynBsc — CHANGES_REQUESTED** [review]
Hey @RomuDeuxfois 👋 

Just the metric cards part not clear to me as we have 4 metrics and not 2, but maybe there is something I haven't grasped properly.

Very clear documentation updates, thanks for that 🚀

File: docs/usage/evaluate/overview.md (hunk 1 above)
Comment (RomuDeuxfois) [reply docs/usage/evaluate/overview.md]: I tried to be cleared

**Comment by RomuDeuxfois** [comment]
I made the changes

**Review by EllynBsc — APPROVED** [review]

**Pull request merged**